.github/actions/zenodo-upload/action.yml

```
@@ -0,0 +1,37 @@
+  steps:
+    - run: pip install -r ${{ github.action_path }}/requirements.txt
+      shell: bash
+    - id: upload
+      run: python ${{ github.action_path }}/zenodo_upload.py
+      shell: bash
+      env:
+        ZENODO_TOKEN: ${{ inputs.token }}
+        ZENODO_FILES: ${{ inputs.files }}
+        ZENODO_CONCEPT_ID: ${{ inputs.concept_id }}
+        ZENODO_METADATA_FILE: ${{ inputs.metadata_file }}
+        ZENODO_SANDBOX: ${{ inputs.sandbox }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)